test/parallel/test-global-this.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
```

Comment:
I am also not sure why we need a test for this as we don't have tests for other JavaScript features.

Reply:
One reason is that it'll block this PR from accidentally getting back-ported to release branches that don't have `globalThis`. But really it's because I figured some extra sanity checks won't hurt.